test/Swashbuckle.AspNetCore.SwaggerGen.Test/SchemaGenerator/JsonSerializerSchemaGeneratorTests.cs

```diff
@@ -1131,6 +1131,17 @@ public void GenerateSchema_HonorsEnumDictionaryKeys_StringEnumConverter()
             Assert.Equal(typeof(IntEnum).GetEnumNames(), referenceSchema.Properties.Keys);
         }
 
+        [Fact]
+        public void GenerateSchema_HonorsEnumDictionaryKeys_StringEnumConverter_CamelCase()
+        {
+            var subject = Subject(null, o => o.DictionaryKeyPolicy = JsonNamingPolicy.CamelCase);
+            var schemaRepository = new SchemaRepository();
+
+            var referenceSchema = subject.GenerateSchema(typeof(Dictionary<IntEnum, string>), schemaRepository);
+
+            Assert.Equal(typeof(IntEnum).GetEnumNames().Select(n => n.ToCamelCase()), referenceSchema.Properties.Keys);
```

Comment:
I would prefer the expected values here are hard-coded, rather than using logic to compute them.